packages/server-core/src/user/user-trade/user-trade.model.ts

```diff
@@ -0,0 +1,41 @@
+import { Sequelize, DataTypes } from 'sequelize'
+import { Application } from '../../../declarations'
+
+export default (app: Application): any => {
```

Comment:
why do we need to store user trade in the database? this should just be a feathers service rather than a feathers-sequelize service & model